templates/home.html

```diff
@@ -1,17 +1,13 @@
 {{ define "content" }}
+  {{ include "partials/flashes" }}
+  
+  <h1 class="sm:text-5xl md:text-6xl">
```

Comment:
These styles must be added to the screen stylesheets if they don't fit into one component

Reply:
Fixed in a6e038f935ccaff5800ed8f2e6ddcfd998182677